yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
why was this edited?

Reply:
generated by yarn, no idea

Reply:
yarn in wrong directory, fixed in 8d98c31